src/redux-state/selectors/realm.ts

```diff
@@ -44,3 +44,7 @@ export const selectDisplayedRealmAddress = selectDisplayedRealmProperty(
 export const selectDisplayedRealmVeTokenAddress = selectDisplayedRealmProperty(
   "veTokenContractAddress"
 )
+
+export const selectNumberOfRealms = createSelector(selectRealms, (realms) =>
+  realms ? Object.entries(realms).length : 0
```

Comment:
Not to code golf, but this could be `Object.entries(realms ?? {})` without losing any generality I think :grin: